actionlist/__tests__/reducers/actionItemsSpec.js

```diff
@@ -212,3 +211,32 @@ describe('action_items', () => {
     expect(reducer(beforeState, action)).toEqual(afterState);
   });
 });
+
+describe('action_items', () => {
+  it('should have the colors reset to white on pressing reset button', () => {
+    const beforeState = {
+      dropdownColors: {
+        Saved: preferenceColors.aqua,
```

Comment:
Isn't there already a `Colors` object in `commons`? What's the difference between that and `preferenceColors` and why does the `actionItems` default state use `Colors` but this test uses `preferenceColors`?